src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -362,22 +388,25 @@ export class AdditionalServiceModalComponent {
                 this.ngProgress.done();
                 this.toggleModal(false);
                 this.reenableButton.emit(false);   
-                this.additionalServicesUtils.navToAdditionalServicePage(this.fromNavbar, this.serviceProcessName);
+                this.additionalServicesUtils.navToAdditionalServicePage(this.fromListPage, this.serviceInfo.process);
                 return;
               }
               return of(o);
             }),
           ))
       ).subscribe(() => {
         this.ngProgress.done();
+        if (status === 'running' || status === 'shutdown') {
+          this.isServiceEnabled = !this.isServiceEnabled;
+        }
```

Comment:
not clear on this

Reply:
Toggle the value of variable after enabling/disabling the service